actors/telemetry-core/src/main/java/org/sunbird/util/lmaxdisruptor/LMAXWriter.java

```diff
@@ -54,14 +55,15 @@ private void init() {
     if (power % 1 != 0) {
       power = Math.ceil(power);
       ringBufferSize = (int) Math.pow(2, power);
-      ProjectLogger.log("New ring buffer size = " + ringBufferSize);
+      ProjectLogger.log(
+          "LMAXWriter:init new ring buffer size = " + ringBufferSize, LoggerEnum.INFO.name());
```

Comment:
Change ...

From:
"LMAXWriter:init new ring buffer size = "

To:
"LMAXWriter:init: New ring buffer size = "